tracer/build/_build/CodeGenerators/CallTargetsGenerator.cs

```diff
@@ -0,0 +1,732 @@
+                string integrationName = null;
+                string typeName = null;
+                string[] typeNames = null;
+                string methodName = null;
+                string returnTypeName = null;
+                string minimumVersion = null;
+                string maximumVersion = null;
+                string[] parameterTypeNames = null;
+                string callTargetType = null;
+                int? integrationKind = null;
+                var instrumentationCategory = InstrumentationCategory.Tracing;
+
+                foreach (var namedArgument in attribute.Properties)
+                {
+                    switch (namedArgument.Name)
+                    {
+                        case nameof(InstrumentAttributeProperties.AssemblyName):
+                            assemblyName = namedArgument.Argument.Value?.ToString();
+                            break;
+                        case nameof(InstrumentAttributeProperties.AssemblyNames):
+                            assemblyNames = GetStringArray(namedArgument.Argument.Value);
+                            break;
+                        case nameof(InstrumentAttributeProperties.IntegrationName):
+                            integrationName = namedArgument.Argument.Value?.ToString();
+                            break;
+                        case nameof(InstrumentAttributeProperties.TypeName):
+                            typeName = namedArgument.Argument.Value?.ToString();
+                            break;
+                        case nameof(InstrumentAttributeProperties.TypeNames):
+                            typeNames = GetStringArray(namedArgument.Argument.Value);
+                            break;
+                        case nameof(InstrumentAttributeProperties.MethodName):
+                            methodName = namedArgument.Argument.Value?.ToString();
+                            break;
+                        case nameof(InstrumentAttributeProperties.ReturnTypeName):
+                            returnTypeName = namedArgument.Argument.Value?.ToString();
+                            break;
+                        case nameof(InstrumentAttributeProperties.MinimumVersion):
+                            minimumVersion = namedArgument.Argument.Value?.ToString();
+                            break;
+                        case nameof(InstrumentAttributeProperties.MaximumVersion):
+                            maximumVersion = namedArgument.Argument.Value?.ToString();
+                            break;
+                        case nameof(InstrumentAttributeProperties.ParameterTypeNames):
+                            parameterTypeNames = GetStringArray(namedArgument.Argument.Value);
+                            break;
+                        case nameof(InstrumentAttributeProperties.CallTargetType):
+                            //callTargetType = (namedArgument.Argument.Value as INamedTypeSymbol)?.ToDisplayString();
```

Comment:
Oh, maybe. I put a breakpoint there that never triggered. Good catch

Reply:
this is originally being used by the ADO.NET instrumentation, are we sure is safe to remove it?

Reply:
@daniel-romano-DD how sure are we this is safe to remove? I can't see a good reason to remove it tbh

We certainly use it in the `GetIntegrationId()` etc calls, but those are used by the source generator so that's ok.

Are we _sure_ it's not used on the native side? 🤔